challenge/public/index.html

```diff
@@ -27,6 +27,10 @@
     <div id="root"></div>
     <h1>Hello World</h1>
     <button id="myButton" onclick="changeColor()">Click Me!</button>
+    <br>
+    <br>
+    <br>
+    <a href="movie.html">Movie</a>
```

Comment:
When redoing this in React, try using a button instead of a link to navigate to the movie page.

Reply:
I will make these changes before the end of this week!